src/services.sh

```diff
@@ -116,4 +116,49 @@ horde::service::rabbitmq() {
 	sleep 3
 }
 
+horde::service::splunk() {
+	local ip=$(horde::bridge_ip)
+	local name="splunk"
+	local hostname="splunk.horde"
+    local logs=$(pwd)
+	local port_cfg="1514:1514"
+
+	horde::delete_stopped splunk || return 1
+
+	horde::cfg_hostname "${hostname}" || return 1
+
+	docker run -d \
+		-p $port_cfg \
+		-p "8000:8000" \
```

Comment:
do we need to expose this port publicly? it looks like it is also configured through fabio at http://splunk.horde